### DISCUSSION TITLE
예약 신청 페이지에서 "예약 신청"버튼은 수동으로 눌러 줘야 하나요?

### DISCUSSION THREAD

**Discussion by dfdfdfffffffff** [changdoc/naver-vaccine-macro#255] [Q&A]
개발자님 안녕하세요.

맵으로 작동 시켰고, 잔여 백신 발생하여 예약신청 페이지까지 자동으로 넘어갔습니다.
아..그런데 "예약 신청"버튼은 수동으로 눌러야 하나요?
아님 자동으로 되는지요?  
페이지까지 넘어 갔는데 마지막 예약 신청버튼이 안눌러졌네요. ㅠㅠ

**Comment by changdoc** [comment]
자동으로 수행할텐데요.
혹시 사용중인 버전이 예전 버전이라면 선택하신 백신 종류와 상관없이 지도에서 예약 신청 페이지로 넘어가는 상황이라 최신 버전으로 사용해보시기 바랍니다.

**Comment by changdoc** [comment]
https://github.com/changdoc/naver-vaccine-macro/discussions/257
여기서 나온 이슈 증상일지도 모르겠네요.

https://github.com/changdoc/naver-vaccine-macro/releases/tag/v1.1.6
1.1.6버전으로 시도해보시면 될 것 같습니다.
